changelog.md

```diff
@@ -1,3 +1,26 @@
+1.9.0
+==========
+### New Features:
+**Microsoft Identity Web now exposes a token provider that the Azure SDKs can use**. See [PR](https://github.com/AzureAD/microsoft-identity-web/pull/542) for details.
+
+**Microsoft Identity Web now supports .NET Framework 4.6.2**. See issue [#1086](https://github.com/AzureAD/microsoft-identity-web/issues/1086).
+
+**Microsoft Identity Web supports calls for regional STS for 1st party only**, this is due to MSAL.NET release [4.29](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/releases/tag/4.29.0), and is available via the `ConfidentialClientApplicationOptions`.
+
+### Bug Fixes:
+**Microsoft Identity Web now locks on the HttpContext, to better handle multi-threaded applications**. See issue [#1097](https://github.com/AzureAD/microsoft-identity-web/issues/1097) and [PR](https://github.com/AzureAD/microsoft-identity-web/pull/1082) and [PR](https://github.com/AzureAD/microsoft-identity-web/pull/1099).
+
+### Fundamentals:
+**Microsoft Identity Web now implements `LoggerMessage` for high performance logging**. See issue [#1105](https://github.com/AzureAD/microsoft-identity-web/issues/1105) for details.
```

Comment:
maybe it could be merged with the performance improvements? Although the error messages are now searchable, so that would deserve indeed its own bullet point?